public/static/docs/api-reference/read.md

```diff
@@ -0,0 +1,88 @@
+# dvc.api.read()
+
+Returns the contents of a tracked file.
+
+## Usage
+
+```py
+import dvc.api
+
+modelpkl = dvc.api.read(
+    'model.pkl',
+    repo='https://github.com/example/project.git'
+    mode='rb')
+```
+
+### Signature
+
+```py
+read(path, repo=None, rev=None, remote=None, mode="r", encoding=None)
+```
+
+### Types
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+The **return** type can be a
+[bytes object](https://docs.python.org/3/glossary.html#term-bytes-like-object)
+or a
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+No exceptions are raised by this function.
+
+## Description
+
+This function wraps [`dvc.api.open()`](/doc/api-reference/open) for a simple and
+direct way to return the complete contents of files tracked in <abbr>DVC
+projects</abbr> (by DVC or Git) – no _context manager_ (`with` keyword) needed.
+
+> Internally, it uses the _file object_'s
+> [`read()`](https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects)
+> method.
+
+## Parameters
+
+- **`path`** - specifies the location of the target data within the source
+  project in `repo`, relative to the project's root.
+
+- `repo` - specifies the location of the source DVC project. If not supplied,
+  defaults to the current DVC project. It can be a URL or a file system path.
+  Both HTTP and SSH protocols are supported for online Git repos (e.g.
+  `[user@]server:project.git`).
+
+  A `dvc.api.UrlNotDvcRepoError` is raised if `repo` is not a valid DVC project.
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, or a commit hash). If not supplied, it uses the default
+  Git revision, `HEAD`. If `repo` is a Git repo, this option is ignored.
+
+- `remote` - name of the [DVC remote](/doc/command-reference/remote) to look for
+  the target data. If not supplied, the cache directory is tried first for local
+  projects; The default remote of `repo` is tried otherwise.
+
+  A `dvc.exceptions.NoRemoteError` is raised if no `remote` is found.
+
+- `mode` - mirrors the namesake parameter in builtin
+  [`open()`](https://docs.python.org/3/library/functions.html#open). Defaults to
+  `"r"` (read).
+
+- `encoding` - used to decode contents to a string. Mirrors the namesake
+  parameter in builtin `open()`. Defaults to `"utf-8"`.
+
+## Examples
```

Comment:
it's still confusing that we have a few examples in open and only one (and without use-case-like name here) - name at least might be helpful

Reply:
read() is pretty simple and all of the more special cases (use or parameters) is covered in open(). When we only have one simple example we just call the section **Examples** – in the cmd ref I mean. But I'm happy to add more examples (basically some of the same ones as in open() but adapted to read()) and/or include example name(s) if you insist.

Reply:
> we just call the section Examples

yeah, it's a bit outdated I think ... better to name them? not sure. Again, up to you on this. No need to put more examples though.

Reply:
OK I guess since we have a basic usage example on top already, naming all examples in api ref makes sense. Done